releasenotes/notes/34962.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+
+# issue is a list of GitHub issues resolved in this note.
+issue:
+- 34838
```

Comment:
This doesn't look right. This should point to an actual issue, not to the failed cherry-pick. If there's none, leave it blank, although, as @jacob-delgado suggested if this PR fixes a bug, we should have a bug filed.